src/index.js

```diff
@@ -56,14 +55,10 @@ module.exports = function(babel) {
           let local = specifierPath.node.local;
           if (local.name !== 'Ember') {
             // Repalce the node with a new `var name = Ember`
```

Comment:
Can you cleanup this comment? What it says is happening isn't right any longer...